lib/screens/explore_screens/vote_screens/proposals/usecases/get_proposals_data_use_case.dart

```diff
@@ -13,14 +13,28 @@ class GetProposalsDataUseCase {
   final ProposalsRepository _proposalsRepository = ProposalsRepository();
 
   Future<List<Result>> run(ProposalType proposalType) {
-    final futures = [
-      _profileRepository.getProfile(settingsStorage.accountName),
-      _proposalsRepository.getProposals(proposalType),
-      _proposalsRepository.getReferendums(proposalType),
-      _proposalsRepository.getSupportLevel(_alliance),
-      _proposalsRepository.getSupportLevel(_campaign),
-      _proposalsRepository.getSupportLevel(_milestone),
-    ];
+    late final List<Future<Result>> futures;
+    if (proposalType.referendumStage.length > 1) {
+      // For History section referendums need 2 request for passed and failed
+      futures = [
+        _profileRepository.getProfile(settingsStorage.accountName),
+        _proposalsRepository.getProposals(proposalType),
+        _proposalsRepository.getReferendums(proposalType.referendumStage.first, proposalType.isReverse),
+        _proposalsRepository.getReferendums(proposalType.referendumStage.last, proposalType.isReverse),
+        _proposalsRepository.getSupportLevel(_alliance),
+        _proposalsRepository.getSupportLevel(_campaign),
+        _proposalsRepository.getSupportLevel(_milestone),
+      ];
+    } else {
+      futures = [
+        _profileRepository.getProfile(settingsStorage.accountName),
+        _proposalsRepository.getProposals(proposalType),
+        _proposalsRepository.getReferendums(proposalType.referendumStage.first, proposalType.isReverse),
+        _proposalsRepository.getSupportLevel(_alliance),
+        _proposalsRepository.getSupportLevel(_campaign),
+        _proposalsRepository.getSupportLevel(_milestone),
```

Comment:
@n13 Now Sorin said that the arrow Its a Unity value (90%).
I don't think we need these get support levels calls anymore. I'll remove them in the next PR.

Reply:
We will need them when we want to show if a proposal is passing or failing, so do not remove them. 

Rather, we need to find where to add that info, but that will be up to @JGDoh - design.